README.md

```diff
@@ -77,6 +78,20 @@ Use `make list-install-targets` to retrieve all the available install targets.
 > CI. Some of this setup runs as the `root` user and **could break your developer's
 > system**. See [Developer Mode](#developer-mode).
 
+### Controlling the CI
+
+#### Github Actions
+
+Kata uses Github Actions in the [kata-containers](https://github.com/kata-containers/kata-containers) repos.
+There are no trigger phrases to invoke these actions as they are triggered by creating a pull request.
```

Comment:
We actually have one trigger phrase used with GitHub actions: `/test_kata_deploy`